mllib-dal/src/main/native/ALSShuffle.cpp

```diff
@@ -76,7 +78,7 @@ Rating *shuffle_all2all(ccl::communicator &comm,
         // perNodeSendLens[i] << endl;
         sendBufSize += perNodeSendLens[i];
     }
-    cout << "sendData size " << sendBufSize << endl;
+    print(INFO, "sendData size %d\n", sendBufSize);
```

Comment:
better to use inside a namespace such as logger::print that it will not be confused with printf.